test/packages/policy_groups/manifest.yml

```diff
@@ -0,0 +1,111 @@
+format_version: 1.0.0
+name: aws
+title: AWS
+version: 0.3.17
+license: basic
+description: AWS Integration
+type: integration
+categories:
```

Comment:
@sorantis Could have a group/set its own categories? Is it always a subset of the package categories?

Reply:
In some cases it's a bigger set, i.e. DynamoDB is an `aws` service, a `cloud` service and a `datastore` service. `datastore` here is specific to DynamoDB and will not be applicable to other, for example compute related services.

Reply:
The question is what happens if someone clicks on a category, what will the filter return:

* All integration sets + package
* Only package
* Only integration sets

Based on the above we should know if we need categories inside the integration sets / policy templates.

Reply:
Yeah and extending to what @ruflin has said, it's more natural to keep DynamoDB, EC2 and others as separate packages ;)

Reply:
If the user click on say "Datastore" then they should be able to see all integration that cover databases, data stores, etc. which also includes cloud services. Since tiles are (will be) generated from policy_templates then I think it's the option 3 in our list. 
This creates another question though. What happens if the user searches for AWS only, do we show AWS as a tile? The answer yes if we create a policy template (integration set) that is called AWS. 

This should work automatically for other integrations, for example NGINX. When the user searches for Nginx we show the Nginx tile because there's a policy template for it, we don't have to use package level information for it.
<img width="501" alt="Screen Shot 2021-02-23 at 11 25 10" src="https://user-images.githubusercontent.com/1685068/108830558-d0fc3d00-75c9-11eb-82c2-66f9cb24453b.png">

Reply:
Speaking of how to express categories in the manifest file, I suggest we keep the top level categories section required, and introduce an optional categories section to a policy template.
This way each policy template will inherit top level categories and also has the option to add more policy template specific categories if needed.
Thoughts? @mtojek @ycombinator @ruflin

Reply:
In other words, categories on the root level (in manifest) would be common buckets valid for all policy templates, right?

Reply:
yes, with the option to extend this list for each policy template. I.e. for AWS common buckets could be `cloud`, `aws` and DynamoDB could optionally extend this list with `datastores` by defining it at the policy template level.